core/src/test/scala/com/evolutiongaming/kafka/flow/RebalanceListenerSpec.scala

```diff
@@ -86,7 +87,7 @@ object RebalanceListenerSpec {
   val partition4 = Partition.unsafe(4)
   val offset = Offset.unsafe(0)
 
-  type F[A] = State[Context, A]
+  type F[A] = StateT[SyncIO, Context, A]
```

Comment:
Why do we want to use `StateT[SyncIO, Context, *]` instead of `State[Context, *]` here? The code under testing, i.e. `RebalanceListener` only requires `Monad`.